src/rpm/tagset.go

```diff
@@ -0,0 +1,384 @@
+package rpm
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/tarantool/cartridge-cli/src/common"
+)
+
+type rpmValueType int32
+
+type rpmTagType struct {
+	ID    int
+	Type  rpmValueType
+	Value interface{}
+}
+
+type rpmTagSetType []rpmTagType
+
+type packedTagType struct {
+	Count int
+	Data  *bytes.Buffer
+}
+
+func (tagSet *rpmTagSetType) addTags(tags ...rpmTagType) {
+	*tagSet = append(*tagSet, tags...)
+}
+
+// Bless @knazarov for writing this code once,
+// I just rewrote it in Go.
+
+// See http://ftp.rpm.org/max-rpm/s1-rpm-file-format-rpm-file-format.html
```

Comment:
Seems it's better to use multiline comments according Tarantool Developers guide:
```go
/*
 *
 *
*/

https://www.tarantool.io/en/doc/2.3/dev_guide/c_style_guide/#commenting-style
It's for C, but anyway
```

Reply:
Changed